manage.py

```diff
@@ -261,6 +261,28 @@ def update_aggregates():
     logger.info('Finished updating incremental aggregates.')
 
 @manager.command
+def add_itemized_partition_cycle(cycle=None, amount=1):
+    """Adds a new itemized cycle child table.
+    By default this will try to add just the current cycle to all partitioned
+    itemized schedule tables if it doesn't already exist.
+    If the child table already exists, skip over it.
+    """
+
+    amount = int(amount)
```

Comment:
Yeah, this is because all of the arguments passed into the management commands come in via strings.  You need to explicitly type cast them (we had been doing this with the `processes` argument elsewhere).